src/test/java/model/TestLottoNumberGenerator.java

```diff
@@ -0,0 +1,17 @@
+package model;
+
+import java.util.List;
+
+class TestLottoNumberGenerator implements view.util.LottoNumberGenerator {
+    private List<Integer> numbers;
```

Comment:
```suggestion
    private final List<Integer> numbers;
```
불변성에 대한 것을 놓치지 않는 것을 습관화하시면 좋을 것 같아요